package.json

```diff
@@ -63,7 +62,7 @@
     "terser": "^5.15.0",
     "ts-jest": "^28.0.8",
```

Comment:
We cannot update `jest` or `jest-environment-jsdom` because `ts-jest` does not support jest v29 yet.